src/lib/tvgCommon.h

```diff
@@ -38,14 +48,12 @@ using namespace tvg;
 
 enum class FileType { Tvg = 0, Svg, Raw, Png, Jpg, Unknown };
 
-//for MSVC Compat
-#ifdef _MSC_VER
-    #define TVG_UNUSED
-    #define strncasecmp _strnicmp
-    #define strcasecmp _stricmp
+#ifdef THORVG_LOG_ENABLED
+    #define TVGLOG(fmt, ...) fprintf(stdout, fmt "\n", ##__VA_ARGS__)  //Error Message for us to fix it
+    #define TVGERR(fmt, ...) fprintf(stderr, fmt "\n", ##__VA_ARGS__)  //Log Message for notifying user some useful hints
```

Comment:
We use module name for most of our logs.
"module: ~~~"
So how about adding a module parameter to the LOG API? or wrapper?
ex)
TVG("~~~");
TVG("LOADER", "~~~");
TVG("SW_ENGINE", "~~~");

Reply:
@JSUYA good suggestion. updated.